dotcom-rendering/src/components/FooterReaderRevenueLinks.stories.tsx

```diff
@@ -0,0 +1,64 @@
+import { css } from '@emotion/react';
```

Comment:
This file replaces `ReaderRevenueLInks.stories.tsx`, and is shorter because it no longer supports the header